galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -709,8 +712,42 @@ def read_settings_from_cache_or_db(
 
         return temp_settings.get(key, value.value)
 
+    def alter_hostname_settings(
+        temp_settings: Settings,
+        value: HookValue,
+        key: str,
+        *args,
+        **kwargs
+    ) -> Any:
+
+        ALLOWED_KEYS = ['CONTENT_ORIGIN', 'ANSIBLE_API_HOSTNAME']
+
+        # if 'CACHE' in key.upper():
+        #    print(f'PID:{os.getpid()} READY:{apps.ready} KEY:{key} VAL:{value.value}')
+
+        if not apps.ready or key.upper() not in DYNAMIC_SETTINGS_SCHEMA:
+            # If app is starting up or key is not on allowed list bypass and just return the value
+            if key.upper() in ALLOWED_KEYS:
+                print(f"SKIPPING {key}")
+            return value.value
```

Comment:
If the only two keys you are interestes are those 2 this part can be simplified to

```suggestion
        if not apps.ready or key.upper() not in ALLOWED_KEYS:
            return value.value
```